### ISSUE TITLE
Multiple tooltips do not work and are not accessible

### ISSUE THREAD

**Issue by valadas** [dnnsoftware/Dnn.React.Common#211]
## Describe the bug
Unfortunately we are still having tooltip issues after #210 so that fix did not work.

## To Reproduce
There are a couple tooltips that do not work, such as Site Settings -> Site Behavior -> Default Pages

I am submitting another PR that completely replaces that component in a few minutes...

**Comment by valadas** [comment]
Closing due to #212 being merged
